src/Bindings/ManualBindings.cpp

```diff
@@ -3297,6 +3297,24 @@ static int tolua_cChunkDesc_GetBlockTypeMeta(lua_State * a_LuaState)
 
 
 
+static int tolua_cColor_GetColor(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamSelf("cColor"))
+	{
+		return 0;
+	}
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
```

Comment:
Right, that was the point of this. :P

Just changed it, it compiles (with gcc, testing with clang now).